templates/default/profile.erb

```diff
@@ -1,48 +1,59 @@
-RDECK_HOME=<%=@rundeck[:basedir]%>
-export RDECK_HOME
-
-RDECK_BASE=<%=@rundeck[:basedir]%>
-export RDECK_BASE
-
-JAVA_CMD=java
-RUNDECK_TEMPDIR=<%=@rundeck[:tempdir]%>
-
+RDECK_BASE="<%=@rundeck[:basedir]%>"
```

Comment:
youre going to have to explain all these changes. I don't think anyone knows what this file does or is supposed to do so I'm not very comfortable with you changing it.